docs/GettingStartedDocs/UseNonDCAPDriverInAcc.md

```diff
@@ -0,0 +1,53 @@
+# Using the Non-DCAP Driver in an ACC VM
```

Comment:
Can we be more specific here. I think it should be 'Using Intel Legacy Launch SGX Driver on ACC Linux VMs'

Reply:
Meta-question: do we want to have the main repo cover docs for all product environments?  (ACC and any other cloud providers in the future)   If yes, that's fine just want to have this agreed on now.  (The alternative would be for ACC and other cloud provider specific stuff to be in one or more separate repos.)

Reply:
if you want to split it out I would prefer to have an ACC directory and have it under there, then other cloud providers have their own directory. Putting in another repo will be harder to find.

Reply:
@dthaler lets leave this in the current directory and create a new issue about the possibility of splitting platform specific documentation out to platform specific directories, such as Azure, On-prem, and any other in the future

Reply:
@paulcallen Sounds good to me.  Created issue #2759